command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
@mrunalp said: `I think that we should require a runtime to implement the CLI otherwise there is no way to test compliance (like @wking said). cc: @RobDolinMS`

Not sure why you guys are saying non client facing runtimes can't be tested for compliance. From my memory, the very reason the runtime spec did not have a client api was because we wanted to allow for other types of OCI products than just clients.  Why wouldn't we want to have interoperability with said other types of clients? What am I missing?